vendor-bundles.js

```diff
@@ -0,0 +1,38 @@
+// Defines a set of vendor bundles which are
+// libraries of 3rd-party code referenced by
+// one or more bundles of Hypothesis client/frontend
+// code
+
+/**
+ * Decorate a module ID with a 'doNotParse' property
+ * indicating that the module bundler should not try
+ * to parse it for require() statements.
```

Comment:
TODO: Add a comment explaining why we would want to disable parsing for a module in the first place. In the case of 'katex', it is because `katex.min.js` is itself a Browserify bundle and consequently contains 'require()' statements which cannot be resolved outside of the original KaTeX source tree. For the other modules, it is because that speeds up the build by avoiding parsing huge JS source files which contain no `require()` statements.